dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -146,8 +148,7 @@ private static void err(String format, Object... args) {
     public static final AtomicBoolean DEBUG = new AtomicBoolean(false);
     public static final AtomicBoolean SCRIPT = new AtomicBoolean(false);
 
-
-    private static boolean isHelp(List<String> args, boolean valOnEmpty) {
+    private static boolean isHelp(java.util.List<String> args, boolean valOnEmpty) {
```

Comment:
Make `java.util.List` the default rather than `scala.collection.immutable.List` (least surprising)